planning/obstacle_stop_planner/include/obstacle_stop_planner/node.hpp

```diff
@@ -121,14 +122,12 @@ class ObstacleStopPlannerNode : public rclcpp::Node
 
   std::unique_ptr<AdaptiveCruiseController> acc_controller_;
   std::shared_ptr<ObstacleStopPlannerDebugNode> debug_ptr_;
-  boost::optional<StopPoint> latest_stop_point_{boost::none};
   boost::optional<SlowDownSection> latest_slow_down_section_{boost::none};
+  std::vector<std::pair<rclcpp::Time, pcl::PointXYZ>> obstacle_history_{};
```

Comment:
making struct would be reasonable and easy to understand.

```
struct ObstacleHistory
{
  rclcpp::Time time
  pcl::PointXYZ point
}
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2647/commits/39ad6e23e50e8608a3bb2dabd956beabfa310650